File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/dto/LoteResponse.java
package com.farmacia.sanidadbackend.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
public class LoteResponse {

    private Long id;

    private String numeroLote;

    private LocalDate fechaFabricacion;

    private LocalDate fechaVencimiento;

    // Datos del proveedor
    private Long proveedorId;

    private String proveedorNombre;

    private String proveedorRuc;

    private String factura;

    private boolean activo;

    private List<LoteDetalleResponse> detalles;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/LoteResponse.kt
package com.sanidad.movil.data.remote.dto

data class LoteResponse(
    val id: Long,
    val numeroLote: String,
    val fechaFabricacion: String?,
    val fechaVencimiento: String?,
    val proveedorId: Long,
    val proveedorNombre: String,
    val proveedorRuc: String,
    val factura: String?,
    val activo: Boolean,
    val detalles: List<LoteDetalleResponse>
)